app/src/components/dashbaord/DashboardTask.tsx

```diff
@@ -0,0 +1,77 @@
+import {Stack, Box, Typography, Button} from '@mui/material';
+import CheckCircleOutlined from '@mui/icons-material/CheckCircleOutlined';
+import LockIcon from '@mui/icons-material/Lock';
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import {SubTask, SubTaskIds} from '../../views/GuestApplicationTracker';
+
+const doSomething = () => {
+  console.log('do something');
+};
+
+interface TaskInfo {
+  title: string;
+  onClick?: () => void;
+}
+
+const taskInfo: {[key in SubTaskIds]: TaskInfo} = {
+  submit_application: {
```

Comment:
As I understand the set of tasks is fixed for the MVP. Is that correct?

Reply:
@paulespinosa I'm not sure about the scope of the tasks for the MVP right now.